shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/spi/ordered/cache/OrderedServicesCacheKey.java

```diff
@@ -17,20 +17,19 @@
 
 package org.apache.shardingsphere.infra.spi.ordered.cache;
 
-import lombok.Getter;
+import lombok.EqualsAndHashCode;
 import lombok.RequiredArgsConstructor;
 
 import java.util.Collection;
-import java.util.Map;
 
 /**
- * Cached ordered services.
+ * Ordered services cache key.
  */
+@EqualsAndHashCode
 @RequiredArgsConstructor
-@Getter
-public final class CachedOrderedServices {
+public class OrderedServicesCacheKey {
```

Comment:
may add final with class

Reply:
How about consider move this class as inner class of `OrderedServicesCache` and simplify the class name as `Key`?

Reply:
> How about consider move this class as inner class of `OrderedServicesCache` and simplify the class name as `Key`?

@terrymanu It's a goods idea, i will optimize it.